users/vieting/jobs/fairseq_labels.py

```diff
@@ -0,0 +1,252 @@
+import soundfile
+import glob
+import os
+import random
+import numpy as np
+import shutil
+import subprocess
+
+from i6_core.lib import corpus
+
+from sisyphus import Job, Task, tk
+
+class FairseqFinetuneCorpusCreationJob(Job):
+    """
+    Creates a corpus directory for finetuning wav2vec with fairseq. This includes the following files:
+    - raw audios files
+    - transcriptions.txt
+    """
+    def __init__(self, corpus_path):
+        """
+        :param [tk.Path]|tk.Path corpus_path: list of paths or single path to corpus xml files.
+        """
+        if isinstance(corpus_path, tk.Path):
+            self.corpus_paths = [corpus_path]
+        else:
+            assert isinstance(corpus_path, list)
+            self.corpus_paths = corpus_path
+        assert min([isinstance(path, tk.Path) for path in self.corpus_paths])
+
+        self.dest_paths = [self.output_path(f"corpus_{i}", directory=True) for i in range(len(self.corpus_paths))]
```

Comment:
```suggestion
        self.out_dest_paths = [self.output_path(f"corpus_{i}", directory=True) for i in range(len(self.corpus_paths))]
```
Please check the contributing guidelines for our recipes: https://github.com/rwth-i6/i6_core/blob/main/CONTRIBUTING.md